README.md

```diff
@@ -64,24 +64,11 @@ Then you can run CeresDB using the default configuration file provided in the co
 ./target/release/ceresdb-server --config ./docs/example.toml
 ```
 
-## Platform Support
```

Comment:
The users may pay special attention to the supported platforms so how about keepping this in the README.md?

Reply:
unrelated issue: "platform" should not be part deploy section and should probably be at the top as well as "deploy"

Reply:
Good references. My point is that our "platform" is not for deployment purposes -- we only support one platform for now. That section is more about how to build and develop CeresDB. We can split "build" and "run" in https://github.com/CeresDB/ceresdb#platform-support and put "run" under "deploy" (maybe only one paragraph in "deploy" I guess).

Reply:
BTW, do you think we need a structured `dev-guide` like this `user-guide`? How about adding a top-level section. Then it will contain `Introduction`, `User Guide`, `Dev Guide`, and `Design`.